client/packages/purchase_orders/src/DetailView/Toolbar.tsx

```diff
@@ -70,26 +71,12 @@ export const Toolbar = ({ isDisabled }: ToolbarProps) => {
             }
           />
         </Grid>
-      </Grid>
-      <Grid
-        display="flex"
-        gap={1}
-        alignItems="flex-end"
-        justifyContent="flex-end"
-        sx={{ marginTop: 1, flexWrap: 'wrap' }}
-      >
-        <Grid display="flex" gap={1} alignItems="flex-end">
-          <SearchBar
-            placeholder={t('placeholder.filter-items')}
-            value={''}
-            onChange={newValue => {
-              // eslint-disable-next-line no-console
-              console.log('TO-DO: Set item filter with:', newValue);
-              // setItemFilter(newValue);
-            }}
-            debounceTime={0}
-          />
-        </Grid>
+        <SearchBar
+          placeholder={t('placeholder.filter-items')}
+          value={itemFilter ?? ''}
+          onChange={newValue => setItemFilter(newValue)}
+          debounceTime={0}
+        />
```

Comment:
I was wondering if it would be helpful to say that it is filtering by code or name?

But I can see that this is the standard string for this filter, so good as gold.

Reply:
Yeah, okay, I've added a comment in the actual filter implentation now, I don't think it matters here

1754798cb5